sandman_web/sandman_web/status/status.py

```diff
@@ -0,0 +1,101 @@
+import psutil, docker, requests
+
+from flask import (
+    Blueprint, render_template
+)
+from werkzeug.exceptions import abort
+
+def process_status(process_name):
+    for process in psutil.process_iter(['pid', 'name']):
+        if process.info['name'] == process_name:
+            return True
+    return False
+
+#Check that Sandman is running
+def check_sandman_health():
+        process_name = "sandman"
+        if process_status(process_name):
+            return True
+        else:
+            return False
+
+#Check that Rhasspy is running and responding
+#Get the Rhasspy contatiner status
+def check_rhasspy_health():
+    client = docker.DockerClient(base_url='unix://var/run/docker.sock')
+    rhasspy_container = client.containers.get("rhasspy")
+    rhasspy_container_status = rhasspy_container.attrs["State"]
+
+    #Get the Rhasspy web response
+    try:
+        rhasspy_web_response = requests.get('http://localhost:12101')
+        rhasspy_web_status = rhasspy_web_response.status_code
+    except:
+        rhasspy_web_status = False
+
+    #Check that the Rhasspy container is running and the web response is OK
+    try:
+        if rhasspy_container_status["Status"] == "running" and rhasspy_web_status == 200:
```

Comment:
I understand what you are saying. Particularly given the function name, returning true or false doesn't easily make sense.

Reply:
This isn't quite what I meant, I meant for example rhasspy_web_status = False isn't really a web status, but something like 404 would be.

Reply:
Okay, I may understand now. You think setting the variable "rhasspy_web_status" to False isn't as good as setting it to 404 given that it otherwise would be a web status code. Is that correct?